src/www/ui/user-edit.php

```diff
@@ -265,6 +265,16 @@ function UpdateUser($UserRec, $SessionIsAdmin)
       $Errors .= "<li>" . _("Invalid email address.") . "</li>";
     }
 
+    /* Make sure email is unique */
+    if (!empty($UserRec['user_email'])) {
+      $email_count = $this->dbManager->getSingleRow(
+        "SELECT COUNT(*) as count FROM users WHERE user_email = $1 LIMIT 1;",
+        array($UserRec['user_email']))["count"];
+    }
+    if ($email_count > 0) {
```

Comment:
`$email_count` can be undefined based on condition if email is empty or not.

Reply:
Fixed it by defining the `$email_count` variable beforehand.